buildenv/travis/build-on-travis.sh

```diff
@@ -111,7 +111,7 @@ if test "x$RUN_BUILD" = "xyes"; then
 
   # Limit number of jobs to work around g++ internal compiler error.
   bash configure --with-freemarker-jar=$TRAVIS_BUILD_DIR/freemarker.jar --with-jobs=$MAKE_JOBS --with-num-cores=$MAKE_JOBS --enable-ccache --with-cmake --disable-ddr
-  make images EXTRA_CMAKE_ARGS="-DOMR_WARNINGS_AS_ERRORS=FALSE"
+  make images EXTRA_CMAKE_ARGS="-DOMR_WARNINGS_AS_ERRORS=FALSE -DCMAKE_C_COMPILER_LAUNCHER=ccache -DCMAKE_CXX_COMPILER_LAUNCHER=ccache"
```

Comment:
Is this required to get ccache?  Can it be connected to the `--enable-ccach` used in the configure line?

Reply:
In theory but I have to do some hackery on the openjdk extensions, because they pass environment variables as part of the ccache command. eg
`CCACHE='CCACHE_DIR=XYZ ccache` which cmake really does not like